packages/@aws-cdk/aws-apigatewayv2/lib/http/route.ts

```diff
@@ -143,6 +160,9 @@ export class HttpRoute extends Resource implements IHttpRoute {
   public readonly httpApi: IHttpApi;
   public readonly path?: string;
 
+  private iamEnabled: boolean = false;
+  private authorizer: IHttpRouteAuthorizer | undefined;
```

Comment:
Why not follow the established pattern of using `?` for optional properties?

```ts
private authorizer?: IHttpRouteAuthorizer;
```

Reply:
I've made your suggested change. :)